docs/annotations/annotations.md

```diff
@@ -12,7 +12,7 @@ The following table documents which sources support which annotations:
 | CloudFoundry |            |          |                   |         |     |                     |
 | CRD          |            |          |                   |         |     |                     |
 | F5           |            |          |                   |         | Yes |                     |
-| Gateway      | Yes        | Yes[^1]  |                   |         | Yes | Yes                 |
+| Gateway      | Yes        | Yes[^1]  |                   | Yes     | Yes | Yes                 |
```

Comment:
Should we footnote to specify which annotations are supported on the `Gateway` and which on the various `*Route` resources? It looks like this PR only supports `target` on the `*Route` resources?